### PR TITLE
Update the selion grid page to expand on all the servlets

### PR DESCRIPTION
**Pull request by ILikeToNguyen**
- List and provide explanation on all the servlets
- Fixed the sporadic unicode dashes to ascii dashes for consistency

### CODE REVIEW

File: html/selionGrid.html [hunk 1]
```diff
@@ -118,23 +106,45 @@ <h2 id="selion-config">SeLion Grid Configuration File</h2>
...
+
+  <div class="container">
+    <img src="../images/grid-management-console.jpg" alt="View of the SeLion Grid Management Console">
+  </div>
+
+  <h3 id="restart nodes">Restarting Nodes Manually</h3>
+  <p>
+    SeLion node processes are automatically restarted after the the number of
+    <code>uniqueSessionCount</code>
+    sessions occur. If you would like to manually force restarting a node, click <u>Force Restart Nodes</u>.
```
Comment (sebady) [inline html/selionGrid.html:141]: s/restarting/restart

**Comment by mach6** [comment]
@sebady you merged this. correct? I see https://github.com/paypal/SeLion/commit/76525b4e46ae4963bb8e0fa81379740806b4718f

**Comment by sebady** [comment]
yes

**Comment by mach6** [comment]
closing. this is merged

**Pull request closed without merging**